orchestra/todos/views.py

```diff
@@ -91,3 +74,17 @@ def perform_update(self, serializer):
             'complete' if todo.completed else 'incomplete')
         message_experts_slack_group(
             todo.task.project.slack_group_id, message)
+
+
+class TodoListTemplateDetail(generics.RetrieveUpdateAPIView):
```

Comment:
do we need the detail view, or do we get the details in the list already?

Reply:
@marcua Retrieve may not be necessary at this point. But we will need to have `Update` to allow designers to save their custom todolist. 

Retrieve may come in handle later on when designers want to check out other designers todolist in a different interface.

Reply:
I want to leave it in. Especially since I have written the tests. :)